api/models/user.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose')
+const { Schema } = mongoose
+
+const User = new Schema(
+    {
+        givenName: { type: String, required: true },
+        familyName: { type: String, required: true },
+        email: { type: String, required: true },
+        imageUrl: String,
+        description: String,
+        supplementaryUrl: String,
```

Comment:
supplementaryUrl can't found on documentation(api/docs/USER.md) example. What's that?

Reply:
This is meant to be the resume or portfolio link for volunteers. Let me know if you have a better suggestion!